neo/rawio/baserawio.py

```diff
@@ -844,6 +844,6 @@ def pprint_vector(vector, lim=8):
         part2 = ' , '.join(e for e in vector[-lim // 2:])
         txt = f"[{part1} ... {part2}]"
     else:
-        part1 = ', '.join(e for e in vector[:lim // 2])
+        part1 = ', '.join(e for e in vector)
```

Comment:
In this case you are only printing the first (by default 4) entries without indicating that there might be still more. Why don't you want to print all entries?

Reply:
I want to print all entries but there were a but for `len(vector) <= lim` only half was printed because of a bad copy/paste.
now it is print entirly.
no ?

Reply:
sorry, I missread the change. My bad. Yes, this makes completely sense!